tsconfig.build.json

```diff
@@ -1,10 +1,11 @@
 {
```

Comment:
I think at this point I can get rid of the tsc build completely and commit to noEmit...